### DISCUSSION TITLE
DragonFly  DB hosted on AWS ECS show (blocked:mixed-content) on js and css pages

### DISCUSSION THREAD

**Discussion by kkalchuri-solutelabs** [dragonflydb/dragonfly#1925] [Q&A]
Hi Team,

I hosted DragonFly DB on my AWS ECS cluster and connected it to an ALB and added a CNAME to it.
While the Dragonfly Index page shows the attached image content only.

On further investigation I found that `status_page.js` and `status_page.css` shows (blocked:mixed-content) error. 
Looks like these pages  are being loaded from a http site.
i.e http://static.dragonflydb.io/data-plane/status_page.css and http://static.dragonflydb.io/data-plane/status_page.js

I am using a [default](docker.dragonflydb.io/dragonflydb/dragonfly) DragonFly DB image in my ECS. 
Is there a way to solve this error and shows the full content of DragonFly 

![image](https://github.com/dragonflydb/dragonfly/assets/124333438/c94c5ba5-5f8e-4979-a894-20a1e7438ea8)

**Comment by chakaz** [comment]
Thank you @kkalchuri-solutelabs, that's a nice catch! I'll fix it shortly, it should be available on the next release

**Reply by chakaz** [reply]
I've created an issue from this discussion, let's continue there:
https://github.com/dragonflydb/dragonfly/issues/1934
